drivers/ethernet/eth_sam_gmac_priv.h

```diff
@@ -257,6 +257,9 @@ struct eth_sam_dev_data {
 	struct device *ptp_clock;
 #endif
 	u8_t mac_addr[6];
+	struct k_timer monitor_timer;
```

Comment:
So we could have here `struct k_delayed_work monitor_work` instead of both k_timer and k_work.